packages/path_provider/path_provider_linux/CHANGELOG.md

```diff
@@ -46,5 +50,5 @@
 * This updates the readme and pubspec and example to reflect the endorsement of this implementation of `path_provider`
 
 ## 0.0.1
-* The initial implementation of path_provider for Linux
+* The initial implementation of path\_provider for Linux
```

Comment:
As a side question, why did we start to include the escape in `path\_provider`?

Reply:
`_` is special markdown syntax (for italics), and one of my editors flags unescaped uses of it as a literal character (outside of ` `` `s), so I've been fixing it as I encounter it. The rules for exactly how the parser decides if an unescaped use is literal or not aren't entirely clear to me, so it seemed like it was better to err or the side of escaping it.